nixos/lib/eval-config-system.nix

```diff
@@ -0,0 +1,56 @@
+# DO NOT IMPORT. Use nixpkgsFlake.lib.nixos, or import (nixpkgs + "/nixos/lib")
+{ lib }: # read -^
+
+let
+
+  # Base: Use the miminal module evaluation
+  evalModulesMinimal = (import ./default.nix {
+    inherit lib;
+    # Implicit use of feature is noted in implementation.
+    featureFlags.minimalModules = { };
+  }).evalModules;
```

Comment:
Was it necessary to move this?
If we leave the minimal evaluation function where it was, that makes the PR easier to understand.

Reply:
This is the legitimate new place of the invocation, `nixos/lib/eval-config.nix` now calls this file as `evalSystemConfiguration` as a dogfooding measure.